packaging/linux/build_binaries.sh

```diff
@@ -94,6 +94,9 @@ build_one_architecture() {
     return
   fi
 
+  # Add the kbfs repo to our custom GOPATH.
+  ln -snf "$kbfs_repo" "$GOPATH/src/github.com/keybase/kbfs"
+
```

Comment:
Just moving this line up from below, so that it's clearer that more than one binary depends on it.